packages/xo-web/src/xo-app/hub/index.js

```diff
@@ -0,0 +1,76 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import { addSubscriptions } from 'utils'
+import { Container, Col, Row } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { isEmpty, map, mapValues, orderBy } from 'lodash'
+import { subscribeResourceCatalog } from 'xo'
+
+import Page from '../page'
+import Resource from './resource'
+
+// ==================================================================
+
+const DEFAULT_SORT_OPTION = {
+  labelId: 'hubSortByName',
+  sortBy: 'name',
+  sortOrder: 'asc',
+}
+
+const HEADER = (
+  <Container>
+    <h2>
+      <Icon icon='menu-hub' /> {_('hubPage')}
+    </h2>
+  </Container>
+)
+
+export default decorate([
+  addSubscriptions({
+    catalog: subscribeResourceCatalog({ filters: { hub: true } }),
+  }),
+  provideState({
+    initialState: () => ({
+      sortBy: DEFAULT_SORT_OPTION.sortBy,
```

Comment:
`sortBy` is static not need to store it in the state.

Reply:
It will be needed in future releases, so I suggest keeping it for now.

Reply:
Why if you don't have a time to do it? So, you can delete it in this PR and add it when it will be needed.